Sources/docc/DocCDocumentation.docc/linking-to-symbols-and-other-content.md

```diff
@@ -0,0 +1,198 @@
+    public var red, green, blue: Double
+}
+
+extension Color {
+    public static let red    = Color(red: 1.0, green: 0.0, blue: 0.0)
+    public static let purple = Color(red: 0.5, green: 0.0, blue: 0.5)
+    public static let blue   = Color(red: 0.0, green: 0.0, blue: 1.0)
+}
+```
+
+Both the `red` property and the `red` static property have a symbol path of 
+`Color/red`. Because these are different types of symbols you can disambiguate 
+`Color/red` with symbol type suffixes instead of the symbols' unique identifiers.
+
+The following example shows a symbol link to the `red` property:
+
+```markdown
+``Color/red-property``
+```
+
+The following example shows a symbol link to the `red` static property:
+
+```markdown
+``Color/red-type.property``
+```
+
+DocC supports the following symbol types for use in symbol links:
+
+| Symbol type       | Suffix            |
+|-------------------|-------------------|
+| Enumeration       | `-enum`           |
+| Enumeration case  | `-enum.case`      |
+| Protocol          | `-protocol`       |
+| Operator          | `-func.op`        |
+| Typealias         | `-typealias`      |
+| Function          | `-func`           |
+| Structure         | `-struct`         |
+| Class             | `-class`          |
+| Type property     | `-type.property`  |
+| Type method       | `-type.method`    |
+| Type subscript    | `-type.subscript` |
+| Property          | `-property`       |
+| Initializer       | `-init`           |
+| Deinitializer     | `-deinit`         |
+| Method            | `-method`         |
+| Subscript         | `-subscript`      |
+| Instance variable | `-ivar`           |
+| Macro             | `-macro`          |
+| Module            | `-module`         |
+
+Symbol type suffixes can include a source language identifier prefix — for 
+example,  `-swift.enum` instead of `-enum`. However, the language 
+identifier doesn't disambiguate the link.
+
+Symbol paths are case-sensitive meaning that symbols with the same name in
+different text casing don't need disambiguation. 
+
+Symbols that have representations in both Swift and Objective-C can use
+symbol paths in either source language. For example, consider a `Sloth` 
+class with `@objc` attributes.
+
+```swift
+@objc public class Sloth: NSObject {
+    @objc public init(name: String, color: Color, power: Power) {
+        self.name = name
+        self.color = color
+        self.power = power
+    }
+}
+```
+
+A symbol link to the Sloth initializer can be written using the symbol 
+path in either source language.
+
+**Swift name**
+
+```markdown
+``Sloth/init(name:color:power:)``
+```
+
+**Objective-C name**
+
+```markdown
+``Sloth/initWithName:color:power:``
+```
+
+### Navigate to an Article
+
+To add a link to an article, use the less-than symbol (`<`), the `doc` keyword, 
+a colon (`:`), the name of the article, and a greater-than symbol 
+(`>`). Don't include the article's file extension in the name. 
+
+```
+<doc:GettingStarted>
```

Comment:
Are filenames case sensitive? For example, are the following different:

`gettingstarted`
`gettingStarted`
`GettingStarted`

???

Reply:
Yes, it has to map exactly to the file name